apps/condo/domains/meter/schema/MeterReading.test.js

```diff
@@ -1011,12 +1043,16 @@ describe('MeterReading', () => {
             const [source] = await MeterReadingSource.getAll(adminClient, { id: CALL_METER_READING_SOURCE_ID })
             const [meterReading] = await createTestMeterReading(client, meter, organization, source)
 
-            await updateTestProperty(adminClient, property.id, { deletedAt: 'true' })
-            const meterReadings = await MeterReading.getAll(client, {
-                id: meterReading.id,
-            })
+            await PropertyAPI.softDelete(adminClient, property.id)
 
-            expect(meterReadings).toHaveLength(0)
+            // NOTE: give worker some time
+            await sleep(1000)
```

Comment:
create your own cases! please, don't change the access tests

Reply:
fixed